releases/release-5.3.0.md

```diff
@@ -0,0 +1,388 @@
+
+    This feature supports TiCDC to replicate incremental data from a TiDB cluster to the secondary relational database TiDB/Aurora/MySQL/MariaDB. In case the primary cluster crashes, TiCDC can recover the secondary cluster to a certain snapshot in the primary cluster within 30 minutes, given the condition that before disaster the replication status of TiCDC is normal and replication lag is small. It allows data loss of less than 5 minutes, that is, RPO <= 30min, and RTO <= 5min.
+
+    [User Document](/ticdc/manage-ticdc.md)
+
+### Deployment and O&M
+
+- **Continuous Profiling (experimental feature)**
+
+    Introduced in TiDB Dashboard, Continuous Profiling allows storing instance performance analysis results automatically in real time without interrupting TiDB clusters. The performance analysis result is displayed using a flame graph, which is observable and shortens troubleshooting time.
+
+    This feature is disabled by default and needs to be enabled on the **Continuous Profile** page of TiDB Dashboard.
+
+    This feature is only available for clusters upgraded or installed using TiUP v1.7.0 or above.
+
+    [User document](/dashboard/continuous-profiling.md)
+
+## Removed feature
+
+Starting from TiCDC v5.3.0, the cyclic replication feature between TiDB clusters (an experimental feature in v5.0.0) has been removed. If you have already used this feature to replicate data before upgrading TiCDC, the related data is not affected after the upgrade.
+
+## Improvements
+
++ TiDB
+
+    - Show the affected SQL statements in the debug log when the coprocessor encounters a lock, which is helpful in diagnosing problems [#27718](https://github.com/pingcap/tidb/issues/27718)
+    - Support showing the size of the backup and restore data when backing up and restoring data in the SQL logical layer [#27247](https://github.com/pingcap/tidb/issues/27247)
+    - Improve the default collection logic of ANALYZE when `tidb_analyze_version=2`, which improves the collection speed and reduces resource overhead
+    - Introduce the `ANALYZE TABLE table_name COLUMNS col_1, col_2, ... , col_n` syntax. The syntax provides a solution for wide tables to collect statistics only on a portion of the columns, which improves the speed of collecting statistics for wide tables.
+
++ TiKV
+
+    - Enhance disk space protection to improve storage stability
+
+        To solve the issue that TiKV might panic when getting the disk fully written error, TiKV introduces a two-level threshold defense mechanism for the disk remaining space to prevent excess traffic from exhausting disk space. Additionally, the mechanism provides the ability to reclaim space when the threshold is triggered. When the remaining space threshold is triggered, some write operations will fail and return a disk full error and a list of disk full nodes. In this case, to recover the space and restore the service, you can execute `Drop/Truncate Table` or scale out the nodes.
```

Comment:
```suggestion
        To solve the issue that TiKV might panic in case of a disk fully-written error, TiKV introduces a two-level threshold defense mechanism to protect the disk remaining space from being exhausted by excess traffic. Additionally, the mechanism provides the ability to reclaim space when the threshold is triggered. When the remaining space threshold is triggered, some write operations will fail and TiKV will return a disk full error as well as a list of disk full nodes. In this case, to recover the space and restore the service, you can execute `Drop/Truncate Table` or scale out the nodes.
```